cms/static/js/spec/models/group_configuration_spec.js

```diff
@@ -149,10 +151,42 @@ define([
             });
 
             it('can pass validation', function() {
+                // Note that two groups - Group A and Group B - are
+                // created by default.
                 var model = new GroupConfigurationModel({ name: 'foo' });
 
                 expect(model.isValid()).toBeTruthy();
             });
+
+            it('requires at least one group', function() {
```

Comment:
Note to reviewers: I moved these test cases from the `GroupModel` suite into the `GroupConfigurationModel` suite, since they have to do with the configuration, not the groups themselves.